examples/preprocessing/plot_interpolate_bad_channels.py

```diff
@@ -22,6 +22,8 @@
 #
 # License: BSD (3-clause)
 
+# sphinx_gallery_thumbnail_number = 2
```

Comment:
the header text of this example needs to be adjusted https://21274-1301584-gh.circle-artifacts.com/0/dev/auto_examples/preprocessing/plot_interpolate_bad_channels.html#sphx-glr-auto-examples-preprocessing-plot-interpolate-bad-channels-py